demo/requirements.txt

```
@@ -0,0 +1,5 @@
+cobs==1.1.4
+libpcap==1.10.0b15
+pyserial==3.5
+python-can==3.3.4
+pyuavcan==1.2.0b1
```

1. You don't need to install dependencies manually. You are supposed to let the PyUAVCAN package pull the required items automatically, that's what package managers are for.
2. Fixing the version number is unlikely to be helpful because the demo application is always versioned synchronously with the library and having to update the version number here after every release (i.e., after every push to master) is tedious and error-prone.

```suggestion
pyuavcan[transport_can_pythoncan,transport_serial,transport_udp]
```